src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -89,7 +89,10 @@
     <Compile Include="$(CommonTestPath)\System\RandomDataGenerator.cs">
       <Link>Common\System\RandomDataGenerator.cs</Link>
     </Compile>
-    <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="Array.1.7.Tests.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.7'">
+    <Compile Include="Array.1.7.Tests.cs" />
```

Comment:
Sorry I missed this eariler but can you rename these to ArrayTests.netstandard1.7.cs and EnumTests.netstandard1.7.cs